app/assets/javascripts/features/navToggle.js

```diff
@@ -0,0 +1,22 @@
+class NavToggle {
+
+	constructor(element) {
+		this.el = element;
+		this.addEvents();
+	}
+
+	addEvents() {
+		console.log(this.el);
+		this.el.addEventListener('click', this.doClick.bind(this));
+	}
+
+	doClick() {
+		console.log('clicked!');
+	}
+};
+
+var elements = document.querySelectorAll("[data-feature='navToggle']");
+for (var i=0; i < elements.length; i++) {
+	var element = elements[i];
+	new NavToggle(element);
+}
```

Comment:
I suppose it's fine to leave this code for now as an example, but make sure it gets deleted later!

I personally like to add comments to my code with "#TODO" in the comment - it makes for an easy text string to search for when you're doing a code clean-up pass later.

Example:
`// #TODO: remove this code later`